engine/src/main/java/org/hibernate/validator/BaseHibernateValidatorConfiguration.java

```diff
@@ -168,6 +168,13 @@
 	@Incubating
 	String CUSTOM_VIOLATION_EXPRESSION_LANGUAGE_FEATURE_LEVEL = "hibernate.validator.custom_violation_expression_language_feature_level";
 
+	/**
+	 * Property to hide values from trace level logging when validating constraints
+	 * @since 6.2
+	 */
+	@Incubating
+	String HIDE_VALIDATION_VALUE_FROM_TRACE_LOGGING = "hibernate.validator.hide_validation_value_from_trace_logging";
```

Comment:
I thought more about it and I think we should hide the values by default.
Could you adjust your patch to make it `show**` (we prefer positive intent) being false by default.